showkase-processor/src/main/java/com/airbnb/android/showkase/processor/logging/ShowkaseValidator.kt

```diff
@@ -39,20 +39,20 @@ internal class ShowkaseValidator {
         }
 
         when {
-            !element.isMethod() -> {
+            !element.isMethod() && !checkElementIsMultiPreview(element) -> {
```

Comment:
Since we return early [here](https://github.com/airbnb/Showkase/pull/255/files#diff-d398a33999c90a9159642dd79319c9723282054253bbbf95b5b75b89d9d5cd9eR106), we shouldn't need any updates in this file no?

Reply:
Might be me that is missing something here, but wouldn't we need to to this check here anyways since the validator is run before that point? 🤔 😃

Reply:
Aha! Thought about this, and it makes sense to do an early return as you said 😄 Didn't see that at first. Changed it up in c57de47  and reverted the validator changes 😄 That was quite clever! Thanks for that!